examples/logger/logging_example.py

```
@@ -79,13 +83,14 @@ def create_key():
     """
     # Generate new key to use with new account
     new_key = PrivateKey.generate()
-    
+
     print(f"Generated new key pair:")
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)